workflow/validate/validate_dag_test.go

```diff
@@ -1039,42 +1039,3 @@ func TestDAGOutputsResolveTaskAggregatedOutputs(t *testing.T) {
 	err := validate(dagOutputsResolveTaskAggregatedOutputs)
 	assert.NoError(t, err)
 }
-
-var dagMissingParamValueInTask = `
```

Comment:
Maybe update the test instead of removing?

Reply:
I reverted my changes as I don't know what thoughts were behind forbidding ValueFrom.Parameter, so I added an additional case for ValueFrom.ConfigMapKeyRef. I also reverted the test and added a new one for the second case.